_config.yml

```diff
@@ -151,24 +155,29 @@ primary_navigation:
 about_us:
   - name: About Us
     url: /about/
-  - name: Why Choose Search.gov?
-    url: "/about/why-choose-searchgov.html"
-  - name: Customers
-    url: "/about/customers.html"
-  - name: Our Product Roadmap
-    url: "/about/updates/roadmap/"
-  - name: Release Notes
-    url: "/about/updates/releases/"
-  - name: Annual Reviews
-    url: "/about/updates/annual-reviews/"
-  - name: Search.gov in Policy
-    url: "/about/policy/"
-  - name: For Developers
-    url: "/developer/"
-  - name: Security and Compliance
-    url: "/about/security.html"
-  - name: Share Your Feedback
-    url: "/about/feedback.html"
+    children: 
+    - name: Why Choose Search.gov?
+      url: "/about/why-choose-searchgov.html"
+    - name: Customers
+      url: "/about/customers.html"
+    - name: Our Product Roadmap
+      url: "/about/updates/roadmap/"
+    - name: Release Notes
+      url: "/about/updates/releases/"
+    - name: Annual Reviews
+      url: "/about/updates/annual-reviews/"
+    - name: Search.gov in Policy
+      url: "/about/policy/"
+    - name: For Developers
+      url: "/developer/"
+    - name: Security and Compliance
+      url: "/about/security.html"
+    - name: Share Your Feedback
+      url: "/about/feedback.html"
+  - name: Related Resources
+    children: 
+    - name: For Developers
```

Comment:
@amanifarooque can you figure out why this isn't indenting properly after the site builds?

Reply:
I just npm installed and now local dev is the same (incorrect) indentation. Seems like the USWDS CSS changed a bit - I just committed a change to force the old styling to keep the indentation for now.